setup.py

```diff
@@ -132,16 +135,8 @@ def setup_package():
         maintainer="Intel",
         maintainer_email="scripting@intel.com",
         description="MKL Support Functions",
-        long_description="""
-            Intel(R) Math Kernel Library (Intel(R) MKL) support functions are
-            subdivided into the following groups according to their purpose:
-                Version Information
-                Threading Control
-                Timing
-                Memory Management
-                Conditional Numerical Reproducibility Control
-                Miscellaneous
-        """,
```

Comment:
It would be good to move the deleted groups description into README.

Reply:
Thank you for review, I have updated this.